### PR TITLE
Feat: setup Pypi

### PR DESCRIPTION
**Pull request by CaioTeixeira95**
- Issue: #10

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+from setuptools import find_packages, setup
+
+with open("README.md", "r") as file:
+    long_description = file.read()
+
+setup(
+    name="django-polaris-wyre",
+    description="A Django Polaris extension that adds Wyre's Custodial Wallet support",
+    long_description=long_description,
+    version="0.1.0",
```
Comment (matpfernandes) [inline setup.py:10]: What do you think about using some of these methods here ?
https://packaging.python.org/guides/single-sourcing-package-version/

**Review by matpfernandes — APPROVED** [review]
LGTM!

**Review by vinibs — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (CaioTeixeira95) [reply setup.py:10]: Sounds good! Do you think that we should add the version (`__version__`) in the `__init__` file of the package?

Comment (matpfernandes) [reply setup.py:10]: Yep, just remember to update the docs about how to update the versions :)

Comment (CaioTeixeira95) [reply setup.py]: @matpfernandes, I've added to `__init__.py` file the version of the package. Could you please see if that's ok for you?

Comment (CaioTeixeira95) [reply setup.py]: In the future, I'll add to `README` how to update the version.

**Review by matpfernandes — APPROVED** [review]
🚀

**Pull request merged**